azure-devops/azext_devops/dev/boards/commands.py

```diff
@@ -27,3 +33,11 @@ def load_work_commands(self, _):
 
         # query commands
         g.command('query', 'query_work_items', table_transformer=transform_work_item_query_result_table_output)
+
+    with self.command_group('boards work-item', command_type=relationsOps) as g:
+        # relation commands
+        g.command('relation-type list', 'get_relation_types_show',
```

Comment:
What other commands are we expecting for "relation-type' group?

Reply:
nothing as of now,, may be if we introduce relation with artifacts then we will have to do something here (not sure at this point of time)

Reply:
Then should we put it inside relation group itself?
az boards work-item relation list-types or something

Reply:
`az boards work-item relation list-types` <== I like this...
@geverghe : Thoughts?

Reply:
Yeah, list-types sounds better in that case.

Reply:
Perfect - makes sense. Need to ensure there are no groups with single commands unless explicitly required.